mods/cnc/weapons/largecaliber.yaml

```diff
@@ -1,50 +1,4 @@
-70mm:
-	ReloadDelay: 30
-	Range: 4c0
-	Report: tnkfire3.aud
-	Projectile: Bullet
-		Image: 120MM
-		Speed: 853
-	Warhead@1Dam: SpreadDamage
-		Spread: 128
-		Damage: 25
-		Versus:
-			None: 25
-			Wood: 75
-			Light: 100
-			Heavy: 90
-		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Crater
-		InvalidTargets: Vehicle, Structure, Wall, Husk, Trees, Creep
-	Warhead@3Eff: CreateEffect
-		Explosions: small_frag
-		ImpactSounds: xplos.aud
-
-105mm:
```

Comment:
This appears to be removing an unused weapon.  Do you know what this was originally used for?